spec.bs

```diff
@@ -185,6 +185,11 @@ This is detectable because it can change the set of fields that are read from th
     |group|["{{GenerateBidInterestGroup/owner}}"].
   1. If |interestGroup|'s [=interest group/owner=] is failure, or its [=url/scheme=] is not
     "`https`", [=exception/throw=] a {{TypeError}}.
+  1. Optionally, [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.

```

Comment:
Is this the best place to bail? Could this happen after (or before?) the first step (permissions policy check)? 

We have the owner from `group[owner]` as soon as we enter into this method, right?.

Reply:
This is right after the step that parses the origin of the owner and checks if it's https. I don't think we can put it before the parsing step.